modules/qutebrowser/hm.nix

```
@@ -1,223 +1,279 @@
 { config, lib, ... }:
 
-with config.stylix.fonts;
 with config.lib.stylix.colors.withHashtag;
+with config.stylix.fonts;
 
 let
   background = base00;
-  secondary-background = base01;
-  selection-background = base03;
+  error = base08;
   foreground = base05;
-  inverted-foreground = base00;
   info = base0B;
+  inverted-foreground = base00;
+  secondary-background = base01;
   secondary-info = base0C;
+  selection-background = base03;
   warning = base0E;
```

I think this makes more sense in its original order rather than alphabetical, since similar colors were grouped together (all the backgrounds, all the foregrounds, then the other colors).

---

In that case, the groups should be explicitly separated by paragraphs:

```suggestion
  background = base00;
  secondary-background = base01;
  selection-background = base03;

  foreground = base05;
  inverted-foreground = base00;

  error = base08;

  info = base0B;
  secondary-info = base0C;

  warning = base0E;
```

Alternatively, the data structure could be enhanced with attribute sets as follows, and the rest of the file should be adapted:

```suggestion
  background = {
    main = base00;
    secondary = base01;
    selection = base03;
  };

  foreground = {
    inverted = base00;
    main = base05;
  };

  error = base08;

  info = {
    main = base0B;
    secondary = base0C;
  };

  warning = base0E;
```

---

Either is fine for now. These variables can be removed when #207 is implemented

---

In that case, let's chose the variant that requires the least amount of effort: https://github.com/danth/stylix/pull/240/commits/c471f4de1ecce6b174807d86534c681e9cf70c3f.